### ISSUE TITLE
Move column type icon from cell to column header

### ISSUE THREAD

**Issue by Vladimir-Va** [gristlabs/grist-core#445]
Hello Gris team.
Grist lacks the ability to have different data types in the same column (and it is right).
Then why do we need to see a data type icon in each cell?
Wouldn't it be better to move this icon to the column heading and display it:
For Table widget - when hovering over a header (as well as a column dropdown icon) in header left side.
For Card widget - permanently on the right side of the row header.

**Comment by paulfitz** [comment]
Hi @Vladimir-Va, do you mean the reference :link: symbol? Or maybe the formula `=` symbol? Just want to make sure I understand your meaning. If I've got it wrong maybe you could include a small screenshot?

**Comment by Vladimir-Va** [comment]
> Hi @Vladimir-Va, do you mean the reference 🔗 symbol? Or maybe the formula `=` symbol?

Hello @paulfitz yes, ref, formula (bcs if we enter to formula edit mode we can see red `=` symdol) and attachment.

**Comment by paulfitz** [comment]
Got it. And the duplication of the symbol is undesirable to you because it takes up space, or because it is visually unappealing?

**Comment by Vladimir-Va** [comment]
> because it takes up space
Yes, I have discovered a way to work from my smartphone with card widget and I noticed it right away. It would also be convenient to show the icons in the empty row to add a new record.

**Comment by DmitryDemidov** [comment]
Gentlemen, good afternoon! When using tables with a lot of columns, the 🔗 and = icons take up space on the screen, we have to make the columns wider, and then they fit on the screen less and have to use horizontal scrolling. It would be amazingly convenient to be able to disable these icons.
